<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoWindow</class>
 <widget class="QWidget" name="infoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <widget class="QLabel" name="filmName">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>401</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>FilmName</string>
       </property>
      </widget>
      <widget class="QLabel" name="poster">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>185</width>
         <height>281</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>185</width>
         <height>281</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="filmDesc">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>401</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string>Film Descriptions</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="trailerTab">
      <attribute name="title">
       <string>Trailer</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
